src/transaction.ts

```diff
@@ -25,6 +27,8 @@ export type IsolationLevel = ValuesOf<typeof IsolationLevel>;
 
 export interface TransactionContext<T extends UserDatabaseClient> extends DBOSContext {
   readonly client: T;
+  getConfiguredClass<C extends InitConfigMethod>(_cls: C): ConfiguredClass<C, Parameters<C['initConfiguration']>[1]>;
```

Comment:
As discussed above, I think we should remove `getConfiguredClass` from transaction context.

Reply:
I'm not convinced one way or the other.

Reply:
How about we cut to just getConfiguredClass, since it is safer, more versatile, and necessary for workflows.  Then we only need to explain one of them, and it's the same on all the contexts.  (I still think the name of the configuration is valuable in transactions/communicators even if only for diagnostics.)